resources/js/components/Common/MessageBox.vue

```diff
@@ -0,0 +1,35 @@
+<script lang="ts" setup>
+import {Dialog, DialogPanel} from '@headlessui/vue'
+import {type MessageBox as MessageBoxProps} from '@/Types/MessageBox'
+import {type VNode} from 'vue'
+
+defineSlots<{
+  title: Node
+  message: VNode
+  buttons: VNode
+}>()
+
+defineProps<MessageBoxProps>()
+</script>
+
+<template>
+  <Dialog :open="isOpen.value" class="relative z-50" @close="close">
```

Comment:
Is this `.value` necessary ?

```suggestion
  <Dialog :open="isOpen" class="relative z-50" @close="close">
```

Reply:
fixed